main.tf

```diff
@@ -122,26 +123,26 @@ resource "aws_route_table" "private_route1" {
     nat_gateway_id = aws_nat_gateway.public1.id
   }
   tags = {
-    Name = "private_route1"
+    Name = "${var.area_code}-private1"
   }
 }
 
 resource "aws_route_table_association" "public0" {
   subnet_id      = aws_subnet.public0.id
-  route_table_id = aws_route_table.public_route.id
+  route_table_id = aws_route_table.public.id
 }
 
 resource "aws_route_table_association" "public1" {
   subnet_id      = aws_subnet.public1.id
-  route_table_id = aws_route_table.public_route.id
+  route_table_id = aws_route_table.public.id
 }
 
 resource "aws_route_table_association" "private0" {
   subnet_id      = aws_subnet.private0.id
-  route_table_id = aws_route_table.private_route0.id
+  route_table_id = aws_route_table.private0.id
 }
 
 resource "aws_route_table_association" "private1" {
   subnet_id      = aws_subnet.private1.id
```

Comment:
You should not mix unrelated changes in a single pull request. If there is a quick fix or optimization that you want to do - always create a separate branch and open a separate pull request.